### PR TITLE
feat: add unit test

### PR DESCRIPTION
**Pull request by fabian4**

### CODE REVIEW

File: src/puppet-walnut.spec.ts [hunk 1]
```diff
@@ -23,3 +27,130 @@ test.skip('PuppetWalnut perfect restart testing', async t => {
     t.pass('perfect restart succeed at #' + n)
   }
 })
+
+test('message parser for text message', async t => {
+  const walnutMessagePayload = {
+    messageId: '4BF4F950-A0B6-4CC3-86B4-5A9580399BCA',
+    messageList: [
+      {
+        contentType: contentType.text,
```
Comment (su-chang) [inline src/puppet-walnut.spec.ts:36]: I think it's better use `string` than `enum` here.

File: src/puppet-walnut.spec.ts [hunk 2]
```diff
@@ -23,3 +27,130 @@ test.skip('PuppetWalnut perfect restart testing', async t => {
...
+        contentText: 'hello world',
+        contentEncoding: contentEncoding.utf8,
+      },
+    ],
+    messageFileSize: 0,
+    messageItem: MessageRawType.text,
```
Comment (su-chang) [inline src/puppet-walnut.spec.ts:42]: ditto

File: src/puppet-walnut.spec.ts [hunk 3]
```diff
@@ -23,3 +27,130 @@ test.skip('PuppetWalnut perfect restart testing', async t => {
...
+    dateTime: '2020-01-17T14:42:20.840+08:00',
+    destinationAddress: 'sip:106500@botplatform.rcs.domain.cn',
+    senderAddress: 'tel:+8617928222350',
+    conversationId: 'XSFDSFDFSAFDSAS^%',
+    contributionId: 'SFF$#REGFY7&^%THT',
+  }
+  const puppet = new PuppetWalnut({
```
Comment (su-chang) [inline src/puppet-walnut.spec.ts:49]: No need to create `PuppetWalnut` instance for all test case.

**Review by su-chang — APPROVED** [review]
LGTM

**Comment by fabian4** [comment]
ping @huan @IronConrad @lijiarui

**Review by IronConrad — APPROVED** [review]

**Comment by fabian4** [comment]
ping @huan

**Review by huan — APPROVED** [review]
LGTM

It's great to see a new unit test!

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**